deposit_contract/tests/contracts/conftest.py

```diff
@@ -63,28 +62,14 @@ def registration_contract(w3, tester):
     return registration_deployed
 
 
-@pytest.fixture(scope="session")
-def chain_start_full_deposit_thresholds():
-    return [randint(1, 5), randint(6, 10), randint(11, 15)]
-
-
 @pytest.fixture(params=[0, 1, 2])
 def modified_registration_contract(
```

Comment:
This fixture can be safely removed. It was used to make testing chain start event easier by modifying `CHAIN_START_FULL_DEPOSIT_THRESHOLD`.